Std/Data/Nat/Lemmas.lean

```diff
@@ -134,85 +134,167 @@ theorem recDiagOn_succ_succ {motive : Nat → Nat → Sort _} (zero_zero : motiv
 /-! ### le/lt -/
 
 theorem ne_of_gt {a b : Nat} (h : b < a) : a ≠ b := (ne_of_lt h).symm
-
-protected theorem le_of_not_le {a b : Nat} : ¬ a ≤ b → b ≤ a := (Nat.le_total a b).resolve_left
-
-protected theorem lt_iff_le_not_le {m n : Nat} : m < n ↔ m ≤ n ∧ ¬ n ≤ m :=
-  ⟨fun h => ⟨Nat.le_of_lt h, Nat.not_le_of_gt h⟩, fun h => Nat.gt_of_not_le h.2⟩
-
-protected theorem lt_iff_le_and_ne {m n : Nat} : m < n ↔ m ≤ n ∧ m ≠ n :=
-  Nat.lt_iff_le_not_le.trans (and_congr_right fun h =>
-    not_congr ⟨Nat.le_antisymm h, fun e => e ▸ Nat.le_refl _⟩)
+theorem ne_of_lt' {a b : Nat} (h : b < a) : a ≠ b := Nat.ne_of_gt h
 
 @[simp] protected theorem not_le {a b : Nat} : ¬ a ≤ b ↔ b < a :=
   ⟨Nat.gt_of_not_le, Nat.not_le_of_gt⟩
 
 @[simp] protected theorem not_lt {a b : Nat} : ¬ a < b ↔ b ≤ a :=
   ⟨Nat.ge_of_not_lt, flip Nat.not_le_of_gt⟩
 
-theorem le_lt_antisymm {n m : Nat} (h₁ : n ≤ m) (h₂ : m < n) : False :=
-  Nat.lt_irrefl n (Nat.lt_of_le_of_lt h₁ h₂)
+protected alias ⟨lt_of_not_ge, _⟩ := Nat.not_le
```

Comment:
```suggestion
protected alias ⟨lt_of_not_ge, _⟩ := Not.le
```
I'm not sure that this is really desirable, but I think that this would allow dot-notation...

Reply:
Damiano, do you understand the new `alias` syntax? The above line is equivalent to
```
/-- Alias of the forward direction of `Nat.not_le`. -/
protected theorem lt_of_not_ge := Nat.not_le.1
```

Reply:
Ah, apparently I do not understand the syntax!

Anyway, I'm not sure that having a declaration `Not.le` around is necessarily a good idea!

Reply:
Where is `Not.le`? I can't find it.

Reply:
Damiano thought you were `Nat.not_le` an alias for `Nat.lt_of_not_ge`, not the other way around.

Reply:
OK. I'll mark this resolved then.